app/views/state_file/questions/nj_spouse_state_id/_nj_spouse.html.erb

```diff
@@ -1,22 +1,15 @@
 <% content_for :info_box do %>
   <div class="reveal">
-    <button class="reveal__button"><%= t(".why_ask_this") %></button>
+    <button href="#" class="reveal__button"><%= t(".helper_heading") %></button>
```

Comment:
[pebble] Why add an href to a button?

Reply:
Copy paste error from another component, but strange that others do this. Removed them from NJ, they still exist in other states